<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f3f6fb;
}

#centralwidget {
    background-color: #ffffff;
    border-radius: 20px;
    margin: 8px;
}

QGroupBox#groupBox {
    border: 1px solid #d0e3ff;
    border-radius: 10px;
    margin-top: 20px;
    background-color: #ffffff;
}

QGroupBox#groupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 4px;
    color: #005fb8;
    font-weight: 600;
}

QLabel {
    color: #2b2b2b;
    font-size: 13px;
}

QLineEdit, QSpinBox {
    border: 1px solid #c8d6f2;
    border-radius: 6px;
    padding: 2px 6px;
    background-color: #fdfdff;
}

QLineEdit:focus, QSpinBox:focus {
    border: 1px solid #0078d4;
    background-color: #ffffff;
}

QPushButton {
    min-height: 28px;
    padding: 4px 14px;
    border-radius: 6px;
    border: none;
    background-color: #0078d4;
    color: white;
    font-size: 13px;
}

QPushButton:hover {
    background-color: #1084e0;
}

QPushButton:pressed {
    background-color: #005a9e;
}

QTableWidget {
    background-color: #ffffff;
    gridline-color: #e0e6f2;
    selection-background-color: #cfe4ff;
    selection-color: #000000;
    border-radius: 8px;
    border: 1px solid #dde3f0;
}

QHeaderView::section {
    background-color: #eef3fb;
    color: #444444;
    padding: 4px;
    border: 1px solid #d0d7e6;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>学生信息</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditId">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>姓名:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>年龄:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>班级:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditClass"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxAge">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdate">
        <property name="text">
         <string>更新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
